requirements/prod.in

```diff
@@ -9,11 +9,10 @@ dj-database-url==0.5.0
 django-allauth==0.46.0
 django-anymail[mailgun]==8.4
 django-hashid-field==3.3.3
-django-js-reverse==0.9.1
+https://github.com/ierror/django-js-reverse/archive/7cab78c4531780ab4b32033d5104ccd5be1a246a.tar.gz
```

Comment:
Hm, right now a typo in those fixtures would not break any tests, but would break production. If we want to ditch this package (and there aren't other similar more maintained options) and switch to a hardcoded list, it'd be great to have some way to test that the list stays accurate if views change. 🤔

Reply:
Hmm, you're right. I did find [django-js-routes](https://github.com/ellmetha/django-js-routes) which had a release back in June of this year. It has the option of [dumping the urls to a static js file](https://github.com/ellmetha/django-js-routes#dumping-the-javascript-routes-resolver) which we can use in the actual frontend and JS tests. Then we could diff the output as part of the Python test suite (like we do with the API schema) to detect outdated urls.

Reply:
I created a card https://trello.com/c/lzRwY7PJ/342-consider-replacing-django-js-reverse